src/functions/template-tags/event.php

```diff
@@ -137,28 +136,12 @@ function tribe_get_event( $event = null, $output = OBJECT, $filter = 'raw', $for
 
 		$cache_key = 'tribe_get_event_' . md5( wp_json_encode( $key_fields ) );
 
-		if ( ! $force ) {
-			$initial_unserialize_callback = ini_get( 'unserialize_callback_func' );
-			// Prevent warning from happening.
-			ini_set( 'unserialize_callback_func', '__return_false' );
-
-			$post = $cache->get( $cache_key, Tribe__Cache_Listener::TRIGGER_SAVE_POST );
+		// Try getting the memoized value.
+		$post = $cache[ $cache_key ];
```

Comment:
Do we need a ?
```
$post = $cache[ $cache_key ] ?? false;
```

Reply:
No, the `Tribe__Cache` object implements the `ArrayAccess` interface, and the `offsetGet` method will call the `Tribe__Cache::get` method that already embeds that check. If the key is not set, it will return `false`. 
The `ArrayAccess` API is the one used to quickly access and set the memoized values.